promql/engine.go

```diff
@@ -1585,8 +1596,13 @@ func (ev *evaluator) eval(ctx context.Context, expr parser.Expr) (parser.Value,
 		// Grouping labels must be sorted (expected both by generateGroupingKey() and aggregation()).
 		sortedGrouping := e.Grouping
 		slices.Sort(sortedGrouping)
+		isDynamicParam := true
```

Comment:
Maybe let's invert this and go with the more common term "const"? E.g. `constParam`. (The "is" in there sounds a bit Java-esque to me.)

Reply:
Done!